builtin/providers/aws/resource_aws_route53_record.go

```diff
@@ -748,3 +749,16 @@ func nilString(s string) *string {
 	}
 	return aws.String(s)
 }
+
+func normalizeAwsAliasName(alias interface{}) string {
```

Comment:
will `alias` here ever _not_ be a string? All the invocations thus far are a `string` that I can see